app/assets/stylesheets/application.css

```diff
@@ -10,11 +10,12 @@
  *
  *= require bootstrap-css/css/bootstrap.min.css
  *= require bootstrap-css/css/bootstrap-responsive.min.css
- *= require nvd3/build/nv.d3.css
+ *= require nvd3/build/nv.d3.min.css
  *= require style
  *= require graph
  *= require comment
  *= require font-awesome
+ *= require spectral-workbench/dist/spectral-workbench
```

Comment:
Hi, this looks good overall! Have you been able to test the v1 capture and spectrum display (i.e. /spectrums/01234) code to ensure that including this line doesn't double-include anything? I would guess if there is a double-include as a result here, it might be OK as it would just overwrite the same namespace (initialization isn't done in these files). However it might be good to go find the explicit script tags in various templates and remove them so that we aren't double-including this file? What do you think?

Thanks, overall this looks awesome! Great work, it's super to see this all happening without any tangles!

Reply:
Thanks a lot for the catch 🙌 !!! I don't see this file included as a script tag anywhere. This might've come from the previous commits 😅

Reply:
Aha, i'm sorry - this is just the CSS, so we won't have a double load issue. I had somehow thought it was JS. This should be all good then. Let's merge!

Reply:
Yes, but I don't think we require the file in the first place because it wasn't included in the old interface as well...